test/mongodb.test.js

```diff
@@ -499,149 +499,162 @@ describe('mongodb', function () {
 
     });
 
-    it('should use $set by default if no operator is supplied', function (done) {
-      User.create({name: 'Al', age: 31, email:'al@strongloop'}, function (err1, createdusers1) {
-        should.not.exist(err1);
-        User.create({name: 'Simon', age: 32,  email:'simon@strongloop'}, function (err2, createdusers2) {
-          should.not.exist(err2);
-          User.create({name: 'Ray', age: 31,  email:'ray@strongloop'}, function (err3, createdusers3) {
-            should.not.exist(err3);
-
-            User.updateAll({name: 'Simon'}, {name: 'Alex'}, function(err, updatedusers) {
-              should.not.exist(err);
-              updatedusers.should.be.equal(1);
+    describe('extended operators', function() {
+      if (process.env.MONGODB_VERSION) {
+        var semver = require('semver');
+        // semver.lt doesn't accept 2.4 (2.4.0 is required)
+        if (semver.satisfies('2.6.0', '>' + process.env.MONGODB_VERSION)) {
```

Comment:
I saw the comment, but it wasn't clear that it was related to reversing the arguments. I understand now.